daml-lf/spec/daml-lf-1.rst

```diff
@@ -1521,6 +1545,15 @@ need to be evaluated further. ::
    ——————————————————————————————————————————————————— ValExpOptionSome
      ⊢ᵥ  'Some' @τ e
 
+     ⊢ᵥ  e₁    ⋯    ⊢ᵥ eₙ
+   ——————————————————————————————————————————————————— ValExpTextMap
+     ⊢ᵥ  [t₁ ↦ e₁; ⋯ ; tₙ ↦ eₙ]
```

Comment:
I guess this is why we switched from `TextLit` to `t`?

Reply:
Indeed.